--- a/webinterface/app.py
+++ b/webinterface/app.py
@@ -7,6 +7,7 @@
 from .analysis.analysis import analysis
 from .general.general import general
 from .documentation.documentation import documentation
+from .ml_detector.ml_detector import ml_detector
 from .utils import get_open_redis_ports_in_order


@@ -65,5 +66,6 @@
     app.register_blueprint(analysis, url_prefix="/analysis")
     app.register_blueprint(general, url_prefix="/general")
     app.register_blueprint(documentation, url_prefix="/documentation")
+    app.register_blueprint(ml_detector, url_prefix="/ml_detector")
     app.run(host="0.0.0.0", port=ConfigParser().web_interface_port)
